Cargo.lock

```diff
@@ -1618,6 +1624,16 @@ dependencies = [
  "thiserror 1.0.69",
 ]
 
+[[package]]
+name = "vm-device"
+version = "0.1.0"
+dependencies = [
+ "anyhow",
+ "serde",
+ "thiserror 2.0.12",
+ "vmm-sys-util",
+]
```

Comment:
Leftovers from earlier experimentation? :o

Reply:
oh, yes. sorry.

Reply:
hmmmm. Rebuilding seems to bring this in. I've no idea who's pulling it in.

Reply:
Ok, figured it.